packages/docs/PropTables/FormPropTable.tsx

```diff
@@ -1,15 +1,14 @@
-import { H2, Text } from '@bigcommerce/big-design';
+import { Panel, Text } from '@bigcommerce/big-design';
 import React from 'react';
 
 import { Code, Prop, PropTable, PropTableWrapper } from '../components';
 
 export const FormErrorPropTable: React.FC<{ id?: string }> = ({ id }) => (
-  <>
-    <H2 id={id}>FormControlError</H2>
+  <Panel header="FormControlError" id={id}>
```

Comment:
Why are we adding a `Panel` to this one but not the other props?

Reply:
These components only inherit native props so they don't use the PropsTable component, thus had to add the panel.

Reply:
Fixed now but used a different approach since the PropList had to be made optional anyway